build/SynologyNAS/toolkit/source/megacmdpkg/INFO.sh

```diff
@@ -3,17 +3,18 @@
 source /pkgscripts/include/pkg_util.sh
 
 package="MEGAcmd"   
-version="0.9.9000"
+version="0.9.9"
 displayname="MEGAcmd"        
 maintainer="Mega NZ"      
 arch="$(pkg_get_unified_platform)"
-description="MEGAcmd command line tool"
```

Comment:
description="MEGAcmd command line tool.  Access your MEGA.nz secure cloud storage account and upload/download files, use its commands in scripts, automatically synchronise folders between your MEGA.nz account and your Synology NAS.  Connect to your NAS via ssh or Putty to use the MEGAcmd commands.  Run mega-help from the command line for documentation, or see our User Guide online."